pom.xml

```diff
@@ -89,7 +89,7 @@
       <dependency>
         <groupId>org.robolectric</groupId>
         <artifactId>android-all</artifactId>
-        <version>5.0.0_r2-robolectric-0</version>
+        <version>5.0.0_r2-robolectric-1</version>
```

Comment:
Hi @erd, @jongerrish. I have successfully separated rendering and non-rendering tests, now travis is passing. In the process, instead of modifying the android-all jar, I am creating a new android jar and directing the classloader to the correct jar based on rendering flag.